annotation/core/models.py

```diff
@@ -71,15 +71,21 @@ def boundary(self):
     def __str__(self):
         return self.body
 
-
+class FeedbackOption(models.Model):
+    """The types of reasons people give for thinking sentence is machine generated text"""
+    shortname = models.CharField(max_length=100)
```

Comment:
Let's force the shortnames to be unique by doing:
```
shortname = models.CharField(max_length=100, primary_key=True)
```

Reply:
fixed